examples/demo-react-native-detox-instruments/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "demo-react-native-detox-instruments",
+  "version": "12.4.1",
+  "private": true,
+  "scripts": {
+    "start": "react-native start",
+    "build:ios": "detox build --configuration ios.sim.debug",
+    "test:ios": "detox test --configuration ios.sim.debug --record-performance all",
+    "e2e:ios": "npm run build:ios && npm run test:ios"
+  },
+  "dependencies": {
+    "react": "16.8.4",
+    "react-native": "0.59.x"
+  },
+  "devDependencies": {
+    "detox": "^12.4.1",
+    "detox-instruments": "1.x.x",
```

Comment:
This is unsupported. Please do not include it on any external project.

Reply:
The only supported method is `brew cask`.

Reply:
I can remove it from the examples folder, but this as a test project

Reply:
It doesn’t matter. Detox Instruments should not be installed from npm. It should be installed as described in the developer guide.